.github/workflows/grid_client_nightly.yml

```diff
@@ -133,3 +133,38 @@ jobs:
           steps.zdb.outcome != 'success' ||
           steps.deleteall.outcome != 'success' 
         run: exit 1
+    
+      - name: Set up GitHub CLI
+        uses: wusatosi/setup-gh@v1.1
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
```

Comment:
Do we need these lines?

Reply:
since we removed the logic for searching for an existing issue we shall remove them
will make the changes now